crates/rune/src/ast/item_struct.rs

```diff
@@ -77,10 +77,7 @@ pub struct Field {
     pub visibility: ast::Visibility,
     /// Name of the field.
     pub name: ast::Ident,
-    /// Colon token for the optional type.
-    #[rune(option)]
-    pub colon: Option<ast::Colon>,
     /// The type.
     #[rune(option)]
-    pub field_type: Option<ast::Ident>,
+    pub field_type: Option<(ast::Colon, ast::Type)>,
```

Comment:
Same deal here, should prefer mimicking syn and calling it `ty`: <https://docs.rs/syn/latest/syn/struct.Field.html>. I would use `type` since everything else is more verbose, but since it is a reserved keyword `ty` is the convention used elsewhere. The `field_` prefix is superfluous since it's already part of the type called `Field`.